lib/cortex/client.rb

```diff
@@ -20,7 +20,7 @@ class Client
     include Cortex::Request
 
     def initialize(hasharg)
-      @base_url = hasharg[:base_url] || 'https://cbcortex.com/api/v1'
+      @base_url = hasharg[:base_url] || 'http://cortex.dev/api/v1'
```

Comment:
I do not like this -- we go from defaulting to prod to defaulting to dev, potentially breaking a lot of use cases. I would say the current version should check if it exists, drop some warning if not, and still default to prod. Next version can make the change without destroying anything.